config/routes.rb

```diff
@@ -58,10 +58,14 @@
         end
       end
 
-      resources :chapters, only: %i[index show], constraints: { id: /\d{2}/ } do
+      resources :chapters, only: %i[index show], constraints: { id: /\d{1,2}/ } do
```

Comment:
I'm not sure I understand this? Chapter short codes are always 2 chars

Reply:
My thinking was it could be handy to support one digit,
also, the section endpoint supports one digit too.

np, just rolled back!